Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -45,6 +46,7 @@ noise::module::ModulePtr TilePartitionerNoise::makeCornerModule(const Corners& c
     combiner.setInputModuleSource(2, central.getOutputModule());
     if (mDebugOutput)
     {
+        setLoggingLevel(logging::Level::Debug);
```

Comment:
The `DebugOutput` configuration option enables writing noise module data to the filesystem, which is slow and requires user input on `stdin`. It should be a separate option from the logging level, which may need to be set to "dev debug" due to some issue which is unrelated to noise modules.